lib/multibuild/manifests.ts

```diff
@@ -47,7 +48,20 @@ export type DockerImageManifest = {
 export function getManifest(
 	modem: Dockermodem,
 	repository: string,
+	dockerOpts: { [key: string]: any } = {}
 ): Promise<DockerImageManifest> {
+	let options = {};
+
+	// If there are valid credentials for the target registry we use them
+	// to avoid being rate-limited by the registry.
```

Comment:
I appreciate the original motivation for the PR is this specific issue, but this is a bug fix really: all interactions with a registry should include relevant credentials and before they weren’t. In that sense I think the comment is redundant.